docs/resources/kms_key.md

```diff
@@ -36,8 +36,9 @@ The following arguments are supported:
 * `pending_days` - (Optional, String) Duration in days after which the key is deleted after destruction of the resource,
   must be between 7 and 1096 days. It doesn't have default value. It only be used when delete a key.
 
-* `is_enabled` - (Optional, Bool) Specifies whether the key is enabled. Defaults to true. Changing this updates the
-  state of existing key.
+* `is_enabled` - (Optional, Bool) Specifies whether the key is enabled. Defaults to true.
+  This field only takes effect when the value of `key_state` is **2** or **3**. In other cases, configuring
+  this field has no effect. Changing this updates the state of existing key.
```

Comment:
1. I think the `In other cases...` is redundant.
2. how can user get the value of `key_state` in the first creation time?